alkos/kernel/CMakeLists.txt

```diff
@@ -15,10 +15,15 @@ include_directories(${SYSROOT}/usr/include)
 include_directories(test)
 
 # Additional test definitions - refer to test/tester.hpp file for value specifics
-# add_compile_definitions(ALKOS_TEST=1)
+ add_compile_definitions(ALKOS_TEST=4)
```

Comment:
Suggestion: use cache to set up a default value, and provide this as argument to cmake so you can avoid changing this variable every single PR

Reply:
This is already removed. Not existing in newer PRs